electrum/wallet.py

```diff
@@ -1964,8 +1964,14 @@ def is_frozen_coin(self, utxo: PartialTxInput) -> bool:
         #       True/False if the user explicitly set it,
         #       None otherwise
         if frozen is None:
```

Comment:
note: if the user explicitly sets the frozen state via the GUI, whatever they set is respected. Also, the GUI does not let the user transition back to the `frozen is None` state (the GUI only allows setting `frozen=True` or `frozen=False`).

I've pushed a separate commit to help testing: https://github.com/spesmilo/electrum/commit/8ccd31fe4952a77f63e506b740c1b32e56d1db60 (on master).
So now using the Console, one can manually transition back to `frozen=None`, e.g.:
```
wallet.set_frozen_state_of_coins(["f4f1bfb718c8f3d94b0af6a6cc8d03cb2c6e7738bc3c8aba70498b9d055128e0:0"], None)
```